### DISCUSSION TITLE
Response from calling tool - Raise Error

### DISCUSSION THREAD

**Discussion by WebsheetPlugin** [microsoft/autogen#2370] [Q&A]
![image](https://github.com/microsoft/autogen/assets/84451738/b2c99a25-5a67-4237-9061-fddcc76762fe)

Is there a possibility of raising the error I get from calling a tool? Curently it's writing it only back to text.

 This would allow me to faster develop the app. I have attached an image of what I have currently. I would prefer it to raise an error.

Maybe there is also a way to decide what kind of error should be raised and what is to be passed back to chat?

**Comment by sonichi** [comment]
The discussion happens in #2421
